tools/cldr-apps/js/src/esm/cldrAnnounce.mjs

```diff
@@ -36,29 +39,28 @@ async function getUnreadCount(setUnreadCount) {
   if (setUnreadCount) {
     callbackSetUnread = setUnreadCount;
   }
-  await refresh(callbackSetData);
+  await refresh(callbackSetData, callbackSetCounts);
 }
 
 /**
  * Refresh the Announcements page and/or unread count
  *
  * @param {Function} viewCallbackSetData the callback function for the Announcements page, or null
+ * @param {Function} viewCallbackSetCounts the callback function for the Announcements page, or null
  *
  * The callback function for setting the data may be null if the Announcements page isn't open and
  * we're only getting the number of unread announcements to display in the main header
  */
-async function refresh(viewCallbackSetData) {
+async function refresh(viewCallbackSetData, viewCallbackSetCounts) {
   if (DISABLE_ANNOUNCEMENTS) {
     return;
   }
   if (viewCallbackSetData) {
-    if (!callbackSetData) {
-      // The Announcements page was just opened, so re-fetch the data immediately regardless
-      // of how recently it was fetched previously (for unread count)
-      schedule.reset();
```

Comment:
This schedule.reset isn't needed here anymore since now AnnouncePanel.vue "created" method calls cldrAnnounce.resetSchedule